Spec/RealtimeClientConnection.swift

```diff
@@ -1859,9 +1870,11 @@ class RealtimeClientConnection: QuickSpec {
                     let options = AblyTests.commonAppSetup()
                     options.autoConnect = false
                     options.authCallback = { tokenParams, callback in
-                        callback(getTestTokenDetails(key: options.key, capability: tokenParams.capability, ttl: tokenParams.ttl), nil)
+                        delay(0) {
```

Comment:
Actually, this is already done: https://github.com/ably/ably-ios/blob/e812cda4bde111b2c55a96d9198c6b4812758665/Source/ARTAuth.m#L254